src/Equinox.MessageDb/MessageDbClient.fs

```diff
@@ -14,8 +14,17 @@ type MdbSyncResult = Written of int64 | ConflictUnknown
 type private Format = ReadOnlyMemory<byte>
 
 module private Json =
-  let private jsonNull = JsonSerializer.SerializeToUtf8Bytes(null)
-  let toArray (m: Format) = if m.IsEmpty then jsonNull else m.ToArray()
+  let private jsonNull = ReadOnlyMemory(JsonSerializer.SerializeToUtf8Bytes(null))
+
+  let parseRow idx (reader: DbDataReader) =
```

Comment:
parse (or Get) - (parseRow made me think it was parsing the whole row)

Reply:
I went with from reader `fromReader`